### PR TITLE
Other auto - See your unicorn joke of the SMU below

### PR DESCRIPTION
**Pull request by classygirl44**
Pull Request Checklist:
- [x] You are able to understand what the code is doing on a basic level with the use of comments
- [x] Code is formatted in a way that isn't confusing
- [x] builds cleanly
- [x] builds *to the robot* cleanly
- [x] Code has been tested on the robot/roadkill/other mechanical setup and completes the task it needs to complete
- [ ] What's the best type of story to tell a runaway unicorn?

### CODE REVIEW

File: src/main/java/frc/robot/Main.java [hunk 1]
```diff
@@ -24,9 +24,9 @@ private Main() {
    * <p>If you change your main robot class, change the parameter type.
    */
   public static void main(String... args) {
-    RobotBase.startRobot(Robot::new);
+    // RobotBase.startRobot(Robot::new);
     // RobotBase.startRobot(VisionTester::new);
-    // RobotBase.startRobot(RobotMercury::new);
+    RobotBase.startRobot(RobotMercury::new);
```
Comment (r-wallis) [inline src/main/java/frc/robot/Main.java]: put it back to submit

File: src/main/java/frc/robot/Robot.java [hunk 2]
```diff
@@ -9,6 +9,8 @@
```
Comment (r-wallis) [inline src/main/java/frc/robot/Robot.java:9]: robot.java shouldn't change - only robotMercury
but the changes that remain now will be needed eventually

**Review by r-wallis — DISMISSED** [review]

File: src/main/java/frc/robot/subsystems/instances/DrivetrainMercury.java [hunk 3]
```diff
@@ -245,4 +248,32 @@ public void setPID(double P, double I, double D) {
...
+    }
+    
+    public int isFwdLimitSwitchClosedLeft() {
+        return leftMotorLeader.isFwdLimitSwitchClosed();
+    }
+
+    public int isRevLimitSwitchClosedLeft() {
+        return leftMotorLeader.isRevLimitSwitchClosed();
+    }
+
```
Comment (Silvery-Sheen) [inline src/main/java/frc/robot/subsystems/instances/DrivetrainMercury.java]: If you take shuffleboard out you really don't need to add these

File: src/main/java/frc/robot/subsystems/instances/DrivetrainMercury.java [hunk 4]
```diff
@@ -110,6 +111,8 @@ public DrivetrainMercury() {
         SmartDashboard.clearPersistent("P");
         SmartDashboard.clearPersistent("I");
         SmartDashboard.clearPersistent("D");*/
+
+        winchEngaged = false;
```
Comment (Silvery-Sheen) [inline src/main/java/frc/robot/subsystems/instances/DrivetrainMercury.java]: yeah no

File: src/main/java/frc/robot/subsystems/instances/DrivetrainMercury.java [hunk 5]
```diff
@@ -23,6 +22,8 @@
     private ADXRS450_Gyro gyro;
     private DifferentialDriveOdometry odometry;
 
+    private boolean winchEngaged;
+
```
Comment (Silvery-Sheen) [inline src/main/java/frc/robot/subsystems/instances/DrivetrainMercury.java]: yeah no

**Review by Silvery-Sheen — CHANGES_REQUESTED** [review]
I made comments to code in DrivetrainMercury.java

**Review by r-wallis — DISMISSED** [review]

**Review by r-wallis — APPROVED** [review]

**Review by Silvery-Sheen — APPROVED** [review]

**Pull request merged**